modules/SampleModule/test/test_samplemodule.cpp

```diff
@@ -0,0 +1,113 @@
+#include "gtest/gtest.h"
+#include "rclcpp/rclcpp.hpp"
+#include "tests/rclcpp_utils.hpp"
+#include "samplemodule.hpp"
+class SampleModuleTest : public ::testing::Test {
+public:
+  static void SetUpTestCase()
+  {
+    rclcpp::init(0, nullptr);
+    
+  }
+
+  static void TearDownTestCase()
+  {
+
+
+    rclcpp::shutdown();
+  }
+
+  protected:
+
+  std::shared_ptr<SampleModule> sampleModule;
+  std::shared_ptr<rclcpp::Node> helperNode;
+  rclcpp::executors::SingleThreadedExecutor executor;
+
+  void SetUp() override
+  {
+    helperNode = rclcpp::Node::make_shared("SampleModuleTestHelper_node");
+    sampleModule = std::make_shared<SampleModule>();
+    executor.add_node(helperNode);
+    executor.add_node(sampleModule);
+  }
+
+  void TearDown() override
+  {
+    executor.remove_node(helperNode);
+    executor.remove_node(sampleModule);
+    helperNode.reset();
+    sampleModule.reset();
+  }
+};
+
+TEST_F(SampleModuleTest, testReceivesAbortMessage){
+  ASSERT_EQ(sampleModule->getAborting(), false);
+}
+
+TEST_F(SampleModuleTest, testGetObjectIds){
```

Comment:
```suggestion
TEST_F(SampleModuleTest, testObjectIdsEmptyOnStartup){
```

Reply:
Naming for other tests can be checked as well, I think that the test name should describe the intended behavior. That way we devs will be able to easily understand what failed in the future when a test report is generated.